### ISSUE TITLE
Does not working in conjunction with cordova-sqlite-storage plugin

### ISSUE THREAD

**Issue by thomson02** [nolanlawson/pouchdb-load#32]
With the cordova-sqlite-storage (v0.7.9) installed this plugin appears to hang (using pouch v4.0).

The 'then' and 'catch' functions are never called. The debugger shows:

          OPEN database: _pouch_database
          File: SQLitePLugin.js Line: 174, Column 15 

          new transaction is waiting for open operation
          File: SQLitePLugin.js Line: 105, Column 17

          DB opened: _pouch_database
          File: SQLitePLugin.js Line: 79, Column 15

          OPEN database: https://user:pass@url/database
          File: SQLitePLugin.js Line: 174, Column 15 
 
          new transaction is waiting for open operation
          File: SQLitePLugin.js Line: 105, Column 17 

I can give a code example if required.

**Comment by nolanlawson** [comment]
Yeah unfortunately the cordova-sqlite-storage plugin is not as stable as it used to be (does not pass the PouchDB test suite anymore). Which device did you run this on?

**Comment by thomson02** [comment]
I was using an industrial device; the Motorola/Zebra TC55 running Android 4.4.3.

**Comment by nolanlawson** [comment]
In that case, try `androidDatabaseImplementation: 2`; it uses an older version of sqlite-storage that's more stable:  http://pouchdb.com/api.html#create_database
